src/state/middleware/scigateway.middleware.tsx

```diff
@@ -169,9 +169,11 @@ export const listenToPlugins = (
 
             // If "instant" is not used, by default we only show "warning" and "error" messages.
             if (pluginMessage.detail.payload.severity !== undefined) {
-              const { severity, message } = pluginMessage.detail.payload;
+              const { severity, message, show } = pluginMessage.detail.payload;
```

Comment:
Could you add a comment just to explain what this is for. Also should the name be more like `showSuccessToasts`/`showAllToasts` as giving a value of false will have no effect on errors. I guess it could even be changed to show only errors when show is `undefined` and otherwise just use that value so its configurable for all, in that case a name change wouldn't be necessary as it would just be an override.